maintenance/projects/js-toolkit/CONTRIBUTING.md

```diff
@@ -70,6 +76,87 @@ This command will download all the dependencies needed by the QA projects contai
 
 Note that the `link-js-toolkit` will move all JS Toolkit dependencies in the QA projects to a `link-js-toolkit` section in the `package.json`. This is to prevent Yarn from trying to download these packages from npmjs.com.
 
+### Testing your local version of JS Toolkit
+
+The normal way to use the JS Toolkit in your projects is by letting `npm` download it from npmjs.com.
+
+However, for that to happen, a version must be released and there are times when you simply want to test what you have modified locally in a clone of this repo. In that case, the setup is a bit more difficult because you need to tell `yarn` to use the local versions of the JS Toolkit packages.
+
+But... we have `link-js-toolkit` for that.
+
+The procedure to setup a `yarn` workspace that uses your local copy of the JS Toolkit is as follows:
+
+1. Create a directory (f.e: `toolkit-workspace`) and run `yarn init -y` to create a `package.json` file on it.
```

Comment:
I don't think people do "f.e" (for example) in English but rather ("eg.", or however you want to write it).

"file on it" → "file in it"